docs/api.rst

```diff
@@ -0,0 +1,267 @@
+| ``build.target``      | **Optional** Target host alias (cpu)-(vendor)-(os)                   |
++-----------------------+----------------------------------------------------------------------+
+| ``download``          | Information about the resulting downloadable archive.                |
++-----------------------+----------------------------------------------------------------------+
+| ``download.url``      | Public archive URL                                                   |
++-----------------------+----------------------------------------------------------------------+
+| ``download.size``     | In Bytes                                                             |
++-----------------------+----------------------------------------------------------------------+
+| ``download.mimetype`` | File type                                                            |
++-----------------------+----------------------------------------------------------------------+
+| ``download.date``     | Publication date                                                     |
++-----------------------+----------------------------------------------------------------------+
+
+The complete JSON schema is available in the collection metadata:
+
+* `${SERVER}/buckets/build-hub/collections/releases <https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases>`_
+
+The records added to the collection will be validated against that schema.
+
+
+More about the release record ID
+================================
+
+.. _release_id:
+
+If you have some information about a release, you might want to guess
+its ID directly in order to fetch the individual record directly.
+
+The unique ID of a release is the following:
+
+.. code-block:: none
+
+    {PRODUCT_NAME}_{CHANNEL}_{VERSION}_{PLATFORM}_{LOCALE}
+
+- ``{PRODUCT_NAME}``: It can be either ``firefox``, ``fennec`` or ``thunderbird``
+- ``{CHANNEL}``: It can be either ``aurora``, ``beta``, ``nightly``, ``nightly-old-id``
+  The channel is not part of the ID for ``release`` and ``esr`` builds
+- ``{VERSION}``: It is the full version of the build. Dots are replaced by ``-`` i.e ``55-0-1``, ``55-1b2``
+  For nightly we use the date and time of the build as a version prefix. i.e: ``2017-06-01-10-02-05_55-0a1``
+- ``{PLATFORM}``: It is the target platform. i.e: ``macosx``, ``android-arm``, ``android-api-15``, ``win32``, ``win64``, ``linux-i386``, etc.
+- ``{LOCALE}``: It is the locale code. i.e ``fr-fr``, ``en-us``
+
+All dots are replaced with dashes and all string are in lowercase.
+
+Here are some example of release ID:
+
+- ``firefox_nightly_2017-05-03-03-02-12_55-0a1_win64_en-us``
+- ``thunderbird_52-0-1_linux-x86_64_en-us``
+- ``firefox_aurora_54-0a2_macosx_en-us``
+- ``firefox_beta_52-0b6_linux-x86_64_en-us``
+- ``firefox_50-0rc1_linux-x86_64_fr``
+- ``firefox_52-0esr_linux-x86_64_en-us``
+- ``fennec_nightly-old-id_2017-05-30-10-01-27_55-0a1_android-api-15_multi``
```

Comment:
I think this whole section probably belongs at the top, along with a little information about what kinds of records are available for a given version (i.e. it seems that there's one record available per version x platform x locale combination, so you'll get several records for a given release).